src/std/env.ab

```diff
@@ -162,3 +162,13 @@ pub fun echo_error(message: Text, exit_code: Num = 1): Null {
     printf("\x1b[1;3;97;41m%s\x1b[0m\n", [message])
     if exit_code > 0 : exit(exit_code)
 }
+
+/// Returns whether or not Bash version is lower than provided version
+/// In a form of an array (ex. [3, 2] for Bash 3.2)
+#[allow_absurd_cast]
+pub fun bash_version_less_than(major: Num, minor: Num): Bool {
```

Comment:
I am okay x.y is used currently. We could append more parameters, as the fix version is required in the future.

Reply:
As far as I'm aware only major and minor versions introduce new features in Bash. The patch versions are only for small bug fixing

Reply:
Mine was just a note. I forgot to mention.